### DISCUSSION TITLE
Option to rebuild stubs/boards when new boards are put on CircuitPython download site.

### DISCUSSION THREAD

**Discussion by wmerkens** [joedevivo/vscode-circuitpython#116] [Ideas]
Currently, I keep a git pull of this repo to rebuild the stubs/boards when I encounter a new board and I use the script build-stubs.sh to create new stubs/boards. I then move them in place in the ext replacing older folders. Would be nice if the ext had a system that you could issue a command and it would do this for you.

**Comment by u8915055** [comment]
Thank you for posting this @wmerkens.   I tried to follow this procedure and i was able to rebuild both boards and stubs and moved them over, however for some reason my C3 board still doesnt show up. I see the C3 board (AI thinker ESP32-C3-32S) in the circuitpython  clone that gets copied over for this script to use, but for some reason it wont build the board files for the C3 board.  Any idea why?  Im very new to this.  Thank you.

**Reply by wmerkens** [reply]
I noticed that some of the ESP32 boards have the same ProductID and VendorID so the list of boards seem to be missing entries but if you check the boards directory and look at the metadata.json, you will see the missing boards.

**Reply by u8915055** [reply]
Thanks very much for your response.  I played around with it a bit.  Sort of getting in the right direction i think.  Just very strange that the script that generates these board definitions, that i assume refers to the circuitpython repo to get those boards, doesnt seem to grab them all.  My AI thinker board is there in CP just not in this extension.   Thanks again.
